apps/blockchain/scripts/generate_blockchain_tests.ex

```diff
@@ -33,9 +33,14 @@ defmodule GenerateBlockchainTests do
             end)
 
           cond do
-            failing != 0 -> log_commented_test(relative_path)
-            passing != 0 -> log_test(relative_path)
-            true -> :ok
+            failing != 0 ->
```

Comment:
Just curious about the reasoning for this formatting change. Do we need it?6

Reply:
@germsvel it looks like new formatter version issue. I returned formatting by hand.